doc/glossary.rst

```diff
@@ -0,0 +1,116 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+********
+Glossary
+********
+
+.. glossary::
+
+    Application launcher
+        The Connect component responsible for discovery and launch of DCC applications,
+        relies on the ftrack event system for communication. The plugin source code
+        can be obtained here: https://github.com/ftrackhq/ftrack-application-launcher.git
+
+    ftrack Python api
+        The supported :term:`Python` Application Programmable Interface for
+        communicating with the ftrack workspace. The source code lives here: https://github.com/ftrackhq/ftrack-python-api
+
+    Client
+        The host counterpart interacting with the user, communicates with the host
+        through the ftrack Event system. Clients are launched from the DCC module
+        through an event, or invoked directly in standalone mode. A client can
+        choose to rely on the :term:`UI` module or run standalone. Example of a
+        client is the Maya publisher panel.
+
+    Connect
+        The ftrack desktop application capable of launching DCC applications,
+        publishing files and plugin management. For ftrack Connect documentation
+        and downloads, please head over here: https://www.ftrack.com/en/portfolio/connect
+        The Connect source code can be obtained here: https://github.com/ftrackhq/ftrack-connect.git
+
+    Connect package
+        The :term:`Connect package` is Connect built and packaged for a certain
+        target platform. typically Windows, Mac OS and Linux. If supplies a default
+        Python runtime for running a Connect as an executable, compiled using cx_freeze.
+        Source code can be found here: https://github.com/ftrackhq/ftrack-connect-package.git
+
+    DCC
+        Digital Content Creation tool, e.g. Maya, 3D Studio Max, Unreal, Blender and
+        so on. Each DCC application has an associated framework plugin, the current ones
+        defined are:
+
+         * Maya; https://github.com/ftrackhq/ftrack-connect-pipeline-maya.git
```

Comment:
@henriknorin-ftrack no need to have repositories defined here, you already have a list of applications.